CondCore/SiStripPlugins/plugins/SiStripPedestals_PayloadInspector.cc

```diff
@@ -116,7 +116,7 @@ namespace {
       auto paramValues = PlotBase::inputParamValues();
       auto ip = paramValues.find("DetIds");
       if (ip != paramValues.end()) {
-        auto input = boost::lexical_cast<std::string>(ip->second);
+        auto input = ip->second;
```

Comment:
So casting is not needed here? (same as in SiPixelFEDChannelContainer_PayloadInspector.cc)